web.py

```diff
@@ -213,6 +214,32 @@ def test_ws():
             flash('Event not yet implemented', 'danger')
     return render_template('test_ws.html', recipient=recipient, event=event, events=events)
 
+@roles_accepted(Role.ROLE_ADMIN, Role.ROLE_FINANCE)
+@app.route('/user_kyc', methods=['GET', 'POST'])
+def user_kyc():
+    email = ''
+    if request.method == 'POST':
+        email = request.form['email']
+        user = User.from_email(db.session, email)
+        if user:
+            kycrequest = KycRequest.from_user(db.session, user)
+            if kycrequest.status == 'completed':
+                token = kycrequest.token
+                pdf = kyc_core.download_pdf_backup(token)
+                if pdf:
+                    response = make_response(pdf)
+                    response.headers.set('Content-Type', 'application/pdf')
+                    return response
+                else:
+                    flash('failed to download pdf', 'danger')
+            elif kycrequest.status == 'created':
```

Comment:
dont use a string literal use the constant `STATUS_CREATED`

Reply:
changed to the above